edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Service/modules/CommonModule.cs

```diff
@@ -47,7 +49,8 @@ public class CommonModule : Module
             string storagePath,
             Option<string> workloadUri,
             TimeSpan scopeCacheRefreshRate,
-            bool persistTokens)
+            bool persistTokens,
+            IList<X509Certificate2> trustBundle)
```

Comment:
Why is this being added here?

Reply:
This will need to eventually be passed to the Authenticator which will have the ability to authenticate X.509 certificates. This is specifically needed so that the X.509 CA auth can be done by EdgeHub.

Reply:
Subsequent PRs will use this in short.